api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/LocationTranslatorImpl.java

```diff
@@ -218,4 +165,83 @@ public org.openmrs.Location getOpenmrsParentLocation(Reference location) {
 		
 		return locationReferenceTranslator.toOpenmrsType(location);
 	}
+	
+	private Map<org.openmrs.Location, List<ContactPoint>> getLocationContactDetails(
+	        Collection<org.openmrs.Location> locations) {
+		final String locationContactPointAttributeType = propertyService
+		        .getGlobalProperty(FhirConstants.LOCATION_CONTACT_POINT_ATTRIBUTE_TYPE);
+		
+		if (locationContactPointAttributeType == null || locationContactPointAttributeType.isEmpty()) {
+			return locations.stream().collect(Collectors.toMap(Function.identity(), ignored -> Collections.emptyList()));
+		}
+		
+		final Map<org.openmrs.Location, List<LocationAttribute>> contactPointAttributes = fhirLocationDao
+		        .getActiveAttributesByLocationsAndAttributeTypeUuid(locations, locationContactPointAttributeType);
+		
+		return contactPointAttributes.entrySet().stream().collect(Collectors.toMap(Map.Entry::getKey,
```

Comment:
Not that this shouldn't work, but it seems like we should have a test to cover this method, as we currently don't.

Reply:
Do you mean `FhirLocationDao#getActiveAttributesByLocationsAndAttributeTypeUuid(...)`?
Or `LocationTranslatorImpl#getLocationContactDetails(...)`?

I've added small unit tests, the codecov tool seems to be satisfied, or you mean the unit test could be better? :)